.github/workflows/go.yml

```diff
@@ -26,11 +26,11 @@ jobs:
 
     - name: Build
       run: go build -v ./...
-      working-directory: server
+      working-directory: cmd/server
```

Comment:
can we run build with `.` here ? it will build both client and server... no ?

Reply:
@PWAlessi let's open an issue here https://github.com/opiproject/opi-spdk-bridge/issues so we don't forget to solve it later